.circleci/config.yml

```
@@ -62,6 +62,9 @@ commands:
       - store_artifacts:
           path: acceptance-tests/tests/build/acceptanceTestLogs
           destination: acceptance-tests-logs
+      - store_artifacts:
+          path: /home/circleci/project/acceptance-tests/tests/core.*
```

```suggestion
          path: /home/circleci/project/acceptance-tests/tests/hs_err_pid.*
```

Core dumps are hard to read, the log is more human readable:

`/home/circleci/project/acceptance-tests/tests/hs_err_pid632.log` is a representative variant.

---

updated